server/command.go

```diff
@@ -264,10 +278,20 @@ func (p *Plugin) executeUnlinkCommand(args *model.CommandArgs) (*model.CommandRe
 		return &model.CommandResponse{}, nil
 	}
 
+	if _, err = p.API.UnshareChannel(link.MattermostChannelID); err != nil {
+		if err2 := p.store.SaveChannelSubscription(*subscription); err2 != nil {
+			p.API.LogDebug("Unable to rollback the subscription deletion", "subscriptionID", subscription.SubscriptionID, "error", err2.Error())
+		}
+		p.API.LogDebug("Unable to unshare the channel", "channelID", link.MattermostChannelID, "error", err.Error())
```

Comment:
If we do keep this, let's log as a warning.
```suggestion
		p.API.LogWarn("Unable to unshare the channel", "channelID", link.MattermostChannelID, "error", err.Error())
```